aiida/cmdline/commands/cmd_profile.py

```diff
@@ -133,69 +135,48 @@ def profile_setdefault(profile):
 
 
 @verdi_profile.command('delete')
-@options.FORCE(help='to skip questions and warnings about loss of data')
-@click.option(
-    '--include-config/--skip-config',
-    default=True,
-    show_default=True,
-    help='Include deletion of entry in configuration file.'
-)
-@click.option(
-    '--include-db/--skip-db',
-    'include_database',
-    default=True,
-    show_default=True,
-    help='Include deletion of associated database.'
-)
+@options.FORCE(help='Skip any prompts for confirmation.')
 @click.option(
-    '--include-db-user/--skip-db-user',
-    'include_database_user',
+    '--delete-storage/--no-delete-storage',
     default=False,
```

Comment:
Since we started the discussion on Discourse, you do agree the default here should be `False`, correct? For sure this is preferable for the `sqlite_zip` backend, and I think in other cases it also seems better to err on the side of caution.

Reply:
Yeah was thinking about this as well, but if you do that and they delete the profile expecting that the data also gets deleted, it might be difficult to do it afterwards. Especially for the `core.psql_dos` because all the details of which database was used and where the repo was is now gone 😅 but maybe this is worse than accidentally removing the data?

Reply:
Wait, it _is_ `False` by default? So there is nothing to change right?

Reply:
I would say accidentally removing the data is worse? 🤔 Figuring out the database and repo directory might still be possible (also e.g. checking the backed up config file, but admittedly that's not very user friendly).

Looking at the current experience:

```
❯ verdi profile delete quicksetup
Warning: Deleting profile `quicksetup`, without the storage.
This operation cannot be undone, are you sure you want to continue? [y/N]: y
Success: Profile `quicksetup` was deleted.
```

Maybe we can:

1. Make the _without_ more explicit, i.e. using capitals or sexy `rich` formatting. Alternatively, maybe this should be an option that is also asked for by default? That would make the user think more if they want to delete the storage or not.
2. Also report details on the storage in case it is not deleted, so the user is aware it is still there and where to find it.

Reply:
> Wait, it is False by default? So there is nothing to change right?

Haha, missed this, probably because we were both typing at the same time ^^. Yes, but since you reported on Discourse that it was true, and @giovannipizzi was asking about it, I figured we should still hash this out here.